src/openlcb/AliasCache.cxx

```diff
@@ -185,9 +185,9 @@ bool AliasCache::retrieve(unsigned entry, NodeID* node, NodeAlias* alias)
 {
     HASSERT(entry < size());
     Metadata* md = pool + entry;
-    if (!md->alias) return false;
-    if (node) *node = md->id;
-    if (alias) *alias = md->alias;
+    if (!md->alias_) return false;
```

Comment:
Should we take the opportunity of touching this to fix the style: All if statements must have braces.

Reply:
All of these style issues are fixed in https://github.com/bakerstu/openmrn/pull/433.